doc/ref_impl/cntt-ri2/chapters/chapter04.md

```diff
@@ -15,7 +15,7 @@
 <a name="4.1"></a>
 ## 4.1 Introduction
 
-This chapter documents the steps to deploy Kubernetes based Reference Implementation (RI-2) according to RA-2. The entire deployment has been tested in OPNFV Labs as a part of the [OPNFV Kuberef project](https://wiki.opnfv.org/display/KUB/Kuberef). The Kuberef project stores all the code needed to deploy RI-2 and hence serves as a reference platform for CNF vendors to develop and test against. Currently, Kuberef only focuses on bare-metal RI2 deployments, but support for other uses cases (running on Packet, other infrastructure providers, etc.) will be added as the development progresses.
+This chapter documents the steps to deploy Kubernetes based Reference Implementation (RI-2) according to RA-2. The entire deployment has been tested in OPNFV Labs as a part of the [OPNFV Kuberef project](https://wiki.opnfv.org/display/KUB/Kuberef). The Kuberef project stores all the code needed to deploy RI-2 and hence serves as a reference platform for CNF vendors to develop and test against. Currently, Kuberef only focuses on bare-metal RI2 deployments, but support for other use cases (running on Packet, other infrastructure providers, etc.) will be added as the development progresses.
```

Comment:
Nit: "Packet" -> "Equinix Metal"

Reply:
Also, consistency across chapters for RA2/RI2/RC2 vs. RA-2/RI-2/RC-2 (could be done in a separate PR)

Reply:
> Also, consistency across chapters for RA2/RI2/RC2 vs. RA-2/RI-2/RC-2 (could be done in a separate PR)

That's a good point - I've observed that these inconsistencies exist at various places throughout the CNTT documentation (not only RI2). I will try to check if there are any specific guidelines for this and fix it in another patch.

Reply:
There are no guidelines yet and I think we should start a wiki page to collect these.

Reply:
> There are no guidelines yet and I think we should start a wiki page to collect these.

I was thinking of opening an issue to address this, but a wiki page could be a good starting point as well. I am not quite sure where within the Anuket wiki the page should exist - Do you have any recommendations?

Reply:
@rihabbanday You should open them within the [Anuket wiki RI2 WS page](https://wiki.anuket.io/display/HOME/RI2+-+Kubernetes+Reference+Implementation). Please see examples of pages under the [RM WS page](https://wiki.anuket.io/display/HOME/RM).

Reply:
@pgoyal01 Thanks for the info. I've noticed that these inconsistencies exist in other WSs as well (at least both RI's, both RC's), and I guess we need to agree on a common way of addressing (RA2/RI2/RC2 or RA-2/RI-2/RC-2) across all documents - I'll try to create a "Guidelines" page under "Anuket Common" as you suggested.